app/views/posts/show.html.erb

```diff
@@ -1 +1,33 @@
-<h1> display certain user posts </h1>
\ No newline at end of file
+<section class="post-section">
+  <% user = User.where(id: params[:user_id])[0] %>
```

Comment:
- [ ]  I'd suggest moving the logic responsible for getting the user data to its corresponding controller action, controller logic should be separated from rendering views. Should be something like this:
```ruby
@user = User.find(params[:id])
```
then you can access `user` directly from the view and render its data.